src/NCronJob/Registry/IInstantJobRegistry.cs

```diff
@@ -22,7 +24,7 @@ public interface IInstantJobRegistry
     /// instantJobRegistry.RunInstantJob&lt;MyJob&gt;(new MyParameterObject { Foo = "Bar" });
     /// </code>
     /// </example>
-    void RunInstantJob<TJob>(object? parameter = null, CancellationToken token = default)
+    void RunInstantJob<TJob>(object? parameter = null, bool forceExecution = false, CancellationToken token = default)
```

Comment:
Technically a breaking change.

Another idea would be to have a seperate method:
```csharp
void ForceRunInstantJob<TJob>(object? parameter = null, CancellationToken token = default);
```

Reply:
good point, didn't want the CancellationToken in the middle, which now is a breaking change.

Reply:
Done. Separated them out into their own methods.